### PR TITLE
Changelog for v2.91.0

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,79 @@ Synapse Changelog
...
+  (`#2630 <https://github.com/vertexproject/synapse/pull/2630>`_)
+- Change the Cell ``iterBackupArchive`` and ``iterNewBackupArchive`` routines
+  to always log exceptiosn they encounter, and report the final log message
+  at the appropriate log level for success and failure.
+  (`#2629 <https://github.com/vertexproject/synapse/pull/2629>`_)
+- When normalizing the ``str`` types, when ``onespace`` is specified, we skip
+  the ``strip`` behavior since it is redundant.
+  (`#2635 <https://github.com/vertexproject/synapse/pull/2625>`_)
+- Log exceptions raised by Cell creation in ``initFromArgv``. Catch
+  ``lmdb.LockError`` when opening a LMDB database and re-rase an exception
```
Comment (infime) [inline CHANGELOG.rst:41]: re-raise

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,6 +5,79 @@ Synapse Changelog
...
+-------------------------
+- Updates to the ``inet`` and ``infotech`` models.
+  (`#2634 <https://github.com/vertexproject/synapse/pull/2634>`_)
+  (`#2644 <https://github.com/vertexproject/synapse/pull/2644>`_)
+
+  ``inet:url``
+    ``inet:url`` now recognizes various ``file:///`` values from RFC 8089.
+
+  ``it:sec:cve``
+    The ``it:sec:cve`` now parses various Unicode dashs and converts them to
```
Comment (Cisphyx) [inline CHANGELOG.rst:21]: Extra "The", or "`it:sec:cve` form"

Comment (Cisphyx) [reply CHANGELOG.rst:21]: also 'dashes' instead of 'dashs'

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,79 @@ Synapse Changelog
...
+  the ``backup:dir`` path is not set.
+  (`#2648 <https://github.com/vertexproject/synapse/pull/2648>`_)
+- Add POSIX advisory locking around the Cell ``cell.guid`` file, to prevent
+  multiple processes from attempting to start a Cell from the same directory.
+  (`#2642 <https://github.com/vertexproject/synapse/pull/2642>`_)
+- Change the default ``SLAB_COMMIT_WARN`` time from 5 seconds to 1 second, in
+  order to more quickly identify slow database writes.
+  (`#2630 <https://github.com/vertexproject/synapse/pull/2630>`_)
+- Change the Cell ``iterBackupArchive`` and ``iterNewBackupArchive`` routines
+  to always log exceptiosn they encounter, and report the final log message
```
Comment (Cisphyx) [inline CHANGELOG.rst:34]: 'exceptiosn'

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,79 @@ Synapse Changelog
...
+  at the appropriate log level for success and failure.
+  (`#2629 <https://github.com/vertexproject/synapse/pull/2629>`_)
+- When normalizing the ``str`` types, when ``onespace`` is specified, we skip
+  the ``strip`` behavior since it is redundant.
+  (`#2635 <https://github.com/vertexproject/synapse/pull/2625>`_)
```
Comment (Cisphyx) [inline CHANGELOG.rst:39]: url is 2625, should be 2635

File: CHANGELOG.rst [hunk 5]
```diff
@@ -5,6 +5,79 @@ Synapse Changelog
...
+  (`#2646 <https://github.com/vertexproject/synapse/pull/2646>`_)
+- Fix an issue with heavy Stormtypes objects where caching was preventing
+  some objects from behaving in a dynamic fashion as they were intended to.
+  (`#2640 <https://github.com/vertexproject/synapse/pull/2640>`_)
+- In norming ``int`` values, when something is outside of the minimum or
+  maximum size of the type, we now include the string representation of the
+  valu instead of the raw value.
+  (`#2643 <https://github.com/vertexproject/synapse/pull/2643>`_)
+- Raise a ``NotReady`` exception when a client attempts to resolve an
+  ``aha://`` URL and there have no been any ``aha`` services registered.
```
Comment (Cisphyx) [inline CHANGELOG.rst:69]: 'not been any' instead of 'no been any'

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**